spec/controllers/jobs_controller_spec.rb

```diff
@@ -92,4 +92,17 @@
       expect(Delayed::Job.find(@running.id)).to be
     end
   end
+
+  describe "POST retry_queued" do
+    before do
+      @queued = Delayed::Job.create()
+      @queued.update_attribute(:attempts, 1)
+      sign_in users(:jane)
+    end
+
+    it "run the queued job" do
+      post :retry_queued
+      expect(@queued.run_at.to_s).to eq(Time.zone.now.to_s)
```

Comment:
Above the `post` line, you should assert that `expect(@queued.run_at.to_s).not_to eq(Time.zone.now.to_s)` (or to `eq` something else) to ensure that a change is happening.